test-configs.yaml

```diff
@@ -1635,16 +1675,56 @@ test_configs:
       - baseline
       - kselftest
 
+  - device_type: imx6qp-sabresd
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6sll-evk
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6sx-sdb
```

Comment:
Baseline OK: https://staging.kernelci.org/test/plan/id/5efdb14c4d025032b548b901/
Upstream in LAVA: https://git.lavasoftware.org/lava/lava/-/blob/master/etc/dispatcher-config/device-types/imx6sx-sdb.jinja2